src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,34 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
-    merged_arr = [0] * elements
+def merge(arrA, arrB):
```

Comment:
Hey Brandon, does this work for this? `merge([8,9,10], [2,21,3,9])`

It seems like if the second array has a large number in it, it might break the algo.